inspectit-oce-core/build.gradle

```diff
@@ -26,9 +26,18 @@ jmh {
     jmhVersion = '1.21'
 }
 
+configurations{
+    opencensus
+}
+
 dependencies {
     compileOnly(
-            project(':inspectit-oce-bootstrap')
+            project(':inspectit-oce-bootstrap'),
+            'io.opencensus:opencensus-api:0.18.0'
```

Comment:
let's extract open census version in the `gradle.properties`, I already see a need for this in the project documentation as well, and in other places..